### PR TITLE
#115 solve sending emails twice

### LINKED ISSUE #115: users reported receiving each email twice
from one of our team members

not sure why I receive every notification twice, my email is xxx@xxx.com, maybe I've registered twice somewhere, could you have a look ?

### PR DESCRIPTION
**Pull request by phash**
called setter for mail list

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
marketing

##### Testing
locally tested

##### Refers/Fixes
Fixes #115

### CODE REVIEW

File: app/marketing/management/commands/expiration.py [hunk 1]
```diff
@@ -46,4 +46,4 @@ def handle(self, *args, **options):
                         email_list.append(fulfiller.fulfiller_email)
                     elif fulfiller.profile and fulfiller.profile.email:
                         email_list.append(fulfiller.profile.email)
-                bounty_expire_warning(b, email_list)
+                bounty_expire_warning(b, set(email_list))
```
Comment (mbeacom) [inline app/marketing/management/commands/expiration.py]: With this approach, you'd have to cast the set back to a list. `(b, list(set(email_list)))` -  Additionally, we might want to investigate performing a `distinct` on `email` for these lists before it makes it to this point.

**Comment by mbeacom** [comment]
@phash I'm going to close this for now. Feel free to re-open the PR if you're good with the review comments and still interested. Thanks!

**Comment by SeironWP** [comment]
Hi, just want to say that i got yesterdays mail twice.

File: app/marketing/management/commands/expiration.py (hunk 1 above)
Comment (yjkimjunior) [reply app/marketing/management/commands/expiration.py]: Something like this?

```
for b in bounties:
  email_list = {}
   if b.bounty_owner_email:
      email_list.add(b.bounty_owner_email)
      for fulfiller in b.fulfillments.all():
            if fulfiller.fulfiller_email:
                  email_list.add(fulfiller.fulfiller_email)
            elif fulfiller.profile and fulfiller.profile.email:
                  email_list.add(fulfiller.profile.email)
       bounty_expire_warning(b, list(email_list))
```

**Pull request closed without merging**